system/services/validation/PresideFieldRuleGenerator.cfc

```diff
@@ -176,6 +176,21 @@ component output="false" singleton=true {
 			ArrayAppend( rules, { fieldName=arguments.fieldName, validator="enum", params={ enum=field.enum, multiple=( IsBoolean( field.multiple ?: "" ) && field.multiple ) } } );
 		}
 
+<<<<<<< Updated upstream
```

Comment:
Looks like some merge conflict artifacts still left in the file @sebduggan

Reply:
Oops... Sorted...